spec/spec_reporter.rb

```diff
@@ -11,7 +11,7 @@ def before_suite(suite)
 
   def record(test)
     # Trim leading "test_dddd_" and replace it with "it "
-    test.name.gsub!(/^test_\d{4}_/, "it ")
+    test.name = test.name.gsub(/^test_\d{4}_/, "it ")
```

Comment:
Strings will be frozen by default which means we can't modify the string `test.name` points to without a deprecation warning.